### ISSUE TITLE
Cant find where a print massage is coming from in the console 

### ISSUE THREAD

**Issue by JK-MITC** [GodotVR/godot_openvr#149]
Hi, I am switching between the VR 3Dcamera and a different free 3Dcamera. When doing so I've used the following code below. When doing this i get the error "OpenXR: No viewport was marked with use_xr, there is no rendered output!". Since this is not a problem I want to remove the print function that spamming my console. Also now that i got your attention, is there a better way to swtich betweena VR camera and a Freecam?

func _input(event):
	if event.is_action_pressed("SwitchCamera"):
		print("swtich Cam")
		match VRCamera.is_current():
			true: 
				get_viewport().use_xr = false
				VRCamera.set_current(false) 
				eCamera.set_current(true)
			false:
				get_viewport().use_xr = true
				VRCamera.set_current(true) 
				eCamera.set_current(false)
	pass

**Comment by vilhalmer** [comment]
I don't think this message is an issue with this library, it is separate from the OpenXR backend. Perhaps OpenXR is mistakenly enabled in the project settings? I made that mistake the first time I used godot-openvr as well. The `use_xr` is still needed to have XRServer running though.
